examples/misc/tickedstroke_demo.py

```diff
@@ -0,0 +1,86 @@
+"""
```

Comment:
Agree this would be better in contours or even lines rather than misc.  You may want to point towards it in the path effects demo as well.

Reply:
I've added additional demos in c7f93760b9 and have cross referenced them from the docs and each other.  This may be overkill, and I may not understand all the conventions for the links.